monitoring/oifs_all_mean_map.py

```diff
@@ -85,4 +85,7 @@ def adjust_metadata(self, map_cube, varname: str):
         # Convert unit to °C if varname is given in K
         if map_cube.units.name == "kelvin":
             map_cube.convert_units("degC")
+        # Convert unit to mm/day if varname is given in kg m-2 s-1
+        if timeseries_cube.units.name == "meter^-2-kilogram-second^-1":
+            timeseries_cube.convert_units("meter^-2-kilogram-day^-1")
```

Comment:
Was that tested? Should it not be `map_cube` that is changed?

Reply:
Sorry, that went too quick, copy-paste. Fixed and tested.